### PR TITLE
single cell qc conda recipe

### PR DESCRIPTION
**Pull request by DouglasAbrams**

### CODE REVIEW

File: single_cell_pipeline_qc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package:
+  name: single_cell_pipeline_qc
+  version: '0.6.14'
```
Comment (diljotgrewal) [inline single_cell_pipeline_qc/meta.yaml]: 0.0.1

**Comment by diljotgrewal** [comment]
rename to:

pseudo_bulk_qc_html_report

**Comment by DouglasAbrams** [comment]
**added wgs cohort qc pckg to this PR

File: wgs_qc_html_report/scripts/plots.R [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    mafsummary = args[5]
+    print(maf_file)
+    print( filtered_maf_file)
+    print( oncoplot_path)
+    print( somatic_interactions)
+    print( mafsummary)
+
+    unfiltmaf = maftools::read.maf(maf=maf_file)
+    filtmaf = maftools::read.maf(maf=filtered_maf_file)
+    print("\n\n\n\n\nherreeee\n\n\n\n\nn\n")
```
Comment (diljotgrewal) [inline wgs_qc_html_report/scripts/plots.R]: remove?

File: wgs_qc_html_report/scripts/plots.R [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    oncoplot(filtmaf, oncoplot_path)
+    print("\n\n\n\n\nherreeee\n\n\n\n\nn\n")
+
+    maf_summary(unfiltmaf, mafsummary)
+    print("\n\n\n\n\nherreeee\n\n\n\n\nn\n")
+
+    somatic_interaction_plot(unfiltmaf, somatic_interactions)
+    print("\n\n\n\n\nherreee44e\n\n\n\n\nn\n")
+
+    # maf_file = "/juno/work/shah/abramsd/CODE/wgs_cohort_qc_TESTRUNS/results/500xy/onco_kb-annotated_maf.maf"
```
Comment (diljotgrewal) [inline wgs_qc_html_report/scripts/plots.R]: remove these too?

File: pseudo_bulk_qc_html_report/scripts/filtermafs.R [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+maf <- data.frame()
+for (f in snakemake@input){
+  print(paste0("Sample: ",str_split(basename(f), "[.]")[[1]][1]))
+  maftemp <- read_delim(f, delim = "\t", guess_max = 10^7, col_types = list(PHENO = "c", SOMATIC = "c")) %>%
+    mutate(id = str_split(basename(f), "[.]")[[1]][1]) %>%
+    filter(t_alt_count > 2)
+  maf <- bind_rows(maf, maftemp)
+}
+
+write_delim(maf, snakemake@output[[1]], delim = "\t")
```
Comment (diljotgrewal) [inline pseudo_bulk_qc_html_report/scripts/filtermafs.R]: is this something snakemake specific?

File: pseudo_bulk_qc_html_report/scripts/mergemafs.R [hunk 5]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env Rscript
+
+library(tidyverse)
+library(data.table)
+args <- commandArgs(TRUE)
+
+input = args[1]
+output = args[2]
+maf = data.table::fread(input)
+# maf = filter(maf, t_alt_count > 2)
```
Comment (diljotgrewal) [inline pseudo_bulk_qc_html_report/scripts/mergemafs.R]: remove

File: pseudo_bulk_qc_html_report/scripts/mutationreport.Rmd [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+```
+
+## Tables of high impact mutations {.tabset .tabset-fade .tabset-pills}
+
+### SNVs
+```{r, echo = FALSE}
+snv %>%
+filter((effect_impact == "HIGH") | ((effect_impact == "MODERATE") & str_detect(effect, "NON_SYNONYMOUS_CODING|STOP_GAINED"))) %>%
+  dplyr::mutate(id2 = unlist(purrr::map(id, ~stringr::str_split(., "_snvs")[[1]][1]))) %>%
+  #mutate(tVAF = round(tVAF, 3)) %>%
```
Comment (diljotgrewal) [inline pseudo_bulk_qc_html_report/scripts/mutationreport.Rmd]: remove

**Pull request merged**